pages/signup/index.tsx

```diff
@@ -45,6 +47,7 @@ const SignUpPage = ({ t }: PageProps): React.ReactElement => {
   const [passwordsIdentical, setPasswordsIdentical] = useState(true);
   const [createUser] = useMutation(ADD_EMPLOYER);
   const [createTalent] = useMutation(ADD_TALENT);
+  const [errors, setErrors] = useState({});
```

Comment:
This comment applies to the whole file actually. We have to be careful because in the current master, we already have this `error and setError` because of the firebase signup stuff.

I suggest:
1. rename this state to sth like `formErrors & setFormErrors` (we should - after merging this PR or after resolving conflicts - rename the firebase error stuff to sth like `authErrors & setAuthErrors`
2. The `handleSubmit` function below gets suuuuper  big. Let's put all that password validation stuff in a function. And then in `handleSubmit` call that function right after `if (formValues.email && formValues.password) {`
3. The error stays there, even after you entered a strong password. First thing to do in the function shd be `setFormErrors(null)` or empty object or how you like it.
4. This is more a question: How do we want to style that error? Same as the auth errors (on top of a page a red bar appears)? We can also do the styling in the next PR.

Reply:
regarding point 4.
Maybe we can just use the error msg from MUI. Don't we already use that on the profile page?

https://material-ui.com/components/text-fields/

![Screenshot 2021-03-21 at 12 50 06](https://user-images.githubusercontent.com/63408795/111903750-11719e00-8a44-11eb-9917-ac9928ef6dc3.png)